docs/API.md

```diff
@@ -1081,15 +1124,38 @@ must have a value greater than 1000 - `SRTO_PEERLATENCY`.
 | -------------------- | ----- | ------- | ------ | ------ | -------- | ------ |
 | `SRTO_SNDSYN`        |       | post    | `bool` | true   | true     | false  |
 
-- **[GET or SET]** - Synchronous (blocking) send mode 
+- **[GET or SET]** - When true, sets blocking mode on writing function, when
+it's not ready to perform the operation. When false ("non-blocking mode"), the
```

Comment:
- **[GET or SET]** - When true, sets blocking mode on a writing function when
it's not ready to perform the operation. When false ("non-blocking mode"), the